test/e2e/manage-type-k8s.sh

```diff
@@ -77,6 +77,12 @@ kubectl --context kind-kubeflex wait --for=condition=Established crd cr1s.synthe
 kubectl --context cp1 create ns e2e
 kubectl --context cp1 wait --for=jsonpath='{.status.phase}'=Active ns/e2e --timeout=120s
 
+:
+: -------------------------------------------------------------------------
+: Test PostCreateHook kubeconfig access
+:
+${SRC_DIR}/test-kubeconfig-access.sh -t k8s
+
```

Comment:
Since this test does not involve `cp1`, it could come after the deletion of `cp1`. That would be better, less resource stress.